dll/Models/Comment.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace dal.Models
+{
+    public class Comment
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
Use ModelBuilder in onModelCreating or custom model mapper called from onModelCreating to set up dal models.